packages/marko-web-algolia-site-search/components/index.marko

```diff
@@ -0,0 +1,153 @@
+import { getAsObject, getAsArray, get } from "@parameter1/base-cms-object-path";
```

Comment:
This is probably too site specific to be included in a generic package... really this package should only provide the middleware and the query wrapper components (for doing the algolia query + the base cms query)